modules/pricer/src/main/java/com/opengamma/strata/pricer/fx/BlackVolatilityFxProvider.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (C) 2015 - present by OpenGamma Inc. and the OpenGamma group of companies
+ * 
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.pricer.fx;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Map;
+
+import org.joda.beans.ImmutableBean;
+
+import com.opengamma.strata.basics.currency.CurrencyPair;
+import com.opengamma.strata.market.sensitivity.FxOptionSensitivity;
+import com.opengamma.strata.market.sensitivity.SurfaceCurrencyParameterSensitivity;
+
+/**
+ * Data provider of volatility for FX options in the lognormal or Black-Scholes model.
+ */
+public interface BlackVolatilityFxProvider
+    extends FxProvider, ImmutableBean {
+
+  /**
+   * Gets the valuation date-time.
+   * 
+   * @return the valuation date-time
+   */
+  public abstract ZonedDateTime getValuationDateTime();
+
+  /**
+   * Returns the Black volatility.
+   * 
+   * @param currencyPair the currency pair
+   * @param expiryDate  the option expiry
+   * @param strike  the option strike
+   * @param forward  the underling forward
+   * @return the volatility
+   */
+  public abstract double getVolatility(CurrencyPair currencyPair, LocalDate expiryDate, double strike, double forward);
```

Comment:
As the expiry is measured in date/time for relativeTime, it should be also for the volatility. The type of the second variable of getVolatility should probably be ZonedDateTime (in one or three variables).